Informatics II/005/Task 3/Patertuck Solution/Task3.c

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
+
+void swap(int *A, int i, int j)
+{
+    int temp = A[i];
+    A[i] = A[j];
+    A[j] = temp;
+}
+
+int partition(int *A, int r, int l){
```

Comment:
it just triggers my autism that right is on the left side and left is on the right side but this is really nitpicky.

Reply:
@Twhining like the way you think :)